packages/components/select-v2/src/useSelect.ts

```diff
@@ -301,7 +301,11 @@ const useSelect = (props: ISelectV2Props, emit: SelectEmitFn) => {
     const padding =
       Number.parseFloat(style.paddingLeft) +
       Number.parseFloat(style.paddingRight)
-    ctx.font = style.font
+    let baseFont = style.font
+    if (!/bold/.test(baseFont)) {
+      baseFont = `bold ${baseFont}`
+    }
+    ctx.font = baseFont
```

Comment:
Perhaps using fontWeight is more accurate, because `style.font` may get many results, the font name may contain `bold`, and the returned `font-weight` may be a number.

Reply:
I have changed the judgement method.